docs/usage/configuration-options.md

```diff
@@ -922,6 +922,12 @@ Example for configuring `docker` auth:
 }
 ```
 
+If multiple `hostRules` match a request, then they will be applied in the following order/priority, where latest takes precedence:
```

Comment:
"Where latest takes precedence" is hard to understand for me, we have a list, that I assume is in the priority order? But then our text says "where latest takes precedence" which seems to imply that the last item on the list has the biggest priority?